src/components/runs/ViewRun.tsx

```diff
@@ -426,20 +425,10 @@ const ViewRun = () => {
                             Code
                           </label>
                           <div className="rounded-md border overflow-hidden">
-                            <SyntaxHighlighter
+                            <CodeBlock
```

Comment:
On this view, when I toggle to show raw, it toggles back on its own after a few seconds.
We are long polling for run status (probably resulting in a re-render).

I actually just tested this and this is existing (bugged) behavior.
So if you have any idea or want to fix here I won't say no 😄 , but it won't block the PR.